bazel/README.md

```diff
@@ -25,8 +25,10 @@ testing purposes. The specific versions of the Envoy dependencies used in this b
 up-to-date with the latest security patches.
 
 1. [Install Bazel](https://bazel.build/versions/master/docs/install.html) in your environment.
-2. `bazel fetch //source/...` to fetch and build all external dependencies. This may take some time.
-2. `bazel build //source/exe:envoy-static` from the Envoy source directory.
+2.  Install external dependencies libtoolize, cmake, and realpath libraries separately.
+3.  Install Golang on your machine. This is required as part of building [BoringSSL](https://boringssl.googlesource.com/boringssl/+/HEAD/BUILDING.md)
```

Comment:
@khgandhi We don't need to assume that. However, adding an example command for Ubuntu in addition to the basic instructions would also probably be helpful for a fairly common case.